src/Workspaces/Core/Desktop/Workspace/Storage/StorageOptions.cs

```diff
@@ -14,6 +9,12 @@ internal static class StorageOptions
         public const string OptionName = "FeatureManager/Storage";
 
         public static readonly Option<StorageDatabase> Database = new Option<StorageDatabase>(
-            OptionName, nameof(Database), defaultValue: StorageDatabase.Esent);
+            OptionName, nameof(Database), defaultValue: StorageDatabase.SQLite);
+
+        /// <summary>
+        /// Threshold to start to use a DB (50MB)
+        /// </summary>
+        public static readonly Option<int> SolutionSizeThreshold = new Option<int>(
+            OptionName, nameof(SolutionSizeThreshold), defaultValue: -1);
```

Comment:
Not part of this change.  set to -1 solely so i can easily test locally.  Will be set to 50MB for final review.